_sass/bootstrap/_alert.scss

```diff
@@ -0,0 +1,51 @@
+//
```

Comment:
I tried fiddling with a bootstrap theme and gem, but nothing worked quit right. Using gems or npm packages doesn't work with github pages either. So  I copies all the sass into the repo. YOLO